framework/src/components/cache/defaults/config.js

```diff
@@ -9,7 +9,7 @@ const defaultConfig = {
 			type: 'string',
 			format: 'ipOrFQDN',
 			env: 'LISK_REDIS_HOST',
-			arg: '-r,--redis',
+			arg: '--redis,-r',
```

Comment:
Why change the order? The usual way to list flags is `<short><comma><blank><long>` i.e. `-r, --redis`.

Reply:
With this PR now long format is necessary and short format is optional. So why have to change this order.

Reply:
> With this PR now long format is necessary and short format is optional.
What does this mean? Are `-r` and `--redis` not equivalent?

Reply:
Yes both are equivalent, but when you user specify `arg` in schema it must provide the long format and can skip the short format. So if I specify `arg: '--redis'` then only long flag option will be available.